apps/frontend/pages/404.js

```diff
@@ -0,0 +1,35 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.papiez}>
```

Comment:
please use more meaningful classNames

Reply:
i named this class like that because i did not have any idea how to name it

Reply:
it's probably "wrapper"
other class names aren't meaningful as well
also, it doesn't make sense to create a separate class for each link